### PR TITLE
expose api which return all database data

### PR DESCRIPTION
**Pull request by fdgod09**
## Why this should be merged
It exposes an api to get all data present in orderbook database

## How this works
example api call 

curl -X POST --data '{
    "jsonrpc":"2.0",
    "id"     :1,
    "method" :"orderbook_getOrderBookData",
    "params" :[]
}' -H 'content-type:application/json;' http://127.0.0.1:9650/ext/bc/2QVZLgXSs9ZhcUWgppD543x4iw5oCSrXtasU7Yns21SL2Mgu87/rpc

## How this was tested
I populated the local database with some data and verified it with the api call above.

### CODE REVIEW

File: plugin/evm/service.go [hunk 1]
```diff
@@ -43,3 +44,11 @@ func (api *SnowmanAPI) IssueBlock(ctx context.Context) error {
 	api.vm.builder.signalTxsReady()
 	return nil
 }
+
```
Comment (shubhamgoyal42) [inline plugin/evm/service.go:47]: let's write this in a separate file? maybe in `plugin/evm/orderbook/service.go` (as per our refactoring plan)

File: plugin/evm/limitorders/memory_database.go [hunk 2]
```diff
@@ -39,15 +39,15 @@ const (
...
+	Market                  Market   `json:"market"`
+	PositionType            string   `json:"position_type"`
+	UserAddress             string   `json:"user_address"`
+	BaseAssetQuantity       *big.Int `json:"base_asset_quantity"`
+	FilledBaseAssetQuantity *big.Int `json:"filled_base_asset_quantity"`
+	Price                   *big.Int `json:"price"`
+	Status                  Status   `json:"status"`
+	Signature               []byte   `json:"signature"`
 	RawOrder                interface{}
 	BlockNumber             *big.Int // block number order was placed on
```
Comment (shubhamgoyal42) [inline plugin/evm/limitorders/memory_database.go:52]: json tag for BlockNumber also?

Comment (fdgod09) [reply plugin/evm/limitorders/memory_database.go]: done

File: plugin/evm/service.go (hunk 1 above)
Comment (fdgod09) [reply plugin/evm/service.go]: refactored code. Please recheck

**Review by shubhamgoyal42 — APPROVED** [review]

**Review by atvanguard — APPROVED** [review]

**Pull request merged**